cmd/tools/genrpcserverinterceptors/server_interceptors.tmpl

```diff
@@ -0,0 +1,35 @@
+{{- /*gotype: go.temporal.io/server/cmd/tools/genrpcserverinterceptors.grpcServerData*/ -}}
+// Code generated by cmd/tools/genrpcserverinterceptors. DO NOT EDIT.
+
+package logtags
+
+import (
+{{- range .Imports}}
+	{{printf "%q" .}}
+{{- end}}
+	"go.temporal.io/server/common/log/tag"
+)
+
+func (wt *WorkflowTags) extractFrom{{.Server}}Message(message any) []tag.Tag {
+	switch r := message.(type) {
+	{{- range .Messages}}
+	case {{.Type}}:
+	{{- if or .TaskTokenGetter .WorkflowIdGetter .RunIdGetter}}
+		{{- if .TaskTokenGetter}}
+		return wt.fromTaskToken(r.{{ .TaskTokenGetter}})
+		{{- else}}
+		return []tag.Tag{
+		{{if .WorkflowIdGetter}}	tag.WorkflowID(r.{{.WorkflowIdGetter}}),
+		{{end -}}
+		{{if .RunIdGetter}}	tag.WorkflowRunID(r.{{.RunIdGetter}}),
+		{{end -}}
+		}
+		{{- end}}
+	{{- else}}
+		return nil
+	{{- end -}}
+	{{- end}}
```

Comment:
I hope it didn't take that long to get all the {{- and -}} correct :flushed:

Reply:
Well, I mostly just copied it from what you had. But yes, spent some time on it :-)